[Docs] Update ASP.NET Core configuration instructions (#654)
elastic/apm-agent-dotnet: docs/setup.asciidoc

}
----

> NOTE: Make sure the `app.UseAllElasticApm(...)` line is the first line in the `Configure` method, otherwise the agent won't be able to properly measure the timing of your requests and potentially complete requests may be missed by the agent. 

Reviewer: ```suggestion
The `app.UseAllElasticApm(...)` line must be the first line in the `Configure` method, otherwise the agent won't be able to properly measure the timing of your requests, and potentially complete requests may be missed by the agent. 
```
